### PR TITLE
Chromium Authors -> Flutter Authors

### PR DESCRIPTION
**Pull request by domesticmouse**
/cc @abd99 re https://github.com/flutter/codelabs/pull/15#issuecomment-675221696

### CODE REVIEW

**Comment by eseidelGoogle** [comment]
@Hixie

**Comment by Hixie** [comment]
That only works if you also update the LICENSE and add an AUTHORS file that mentions the Flutter authors, etc. See the flutter/flutter repo for an example.

**Comment by domesticmouse** [comment]
@Hixie I've added an AUTHORS file and added a Flutter Authors line to LICENSE. PTAL

File: LICENSE [hunk 1]
```diff
@@ -1,4 +1,5 @@
 // Copyright 2014,2019 The Chromium Authors. All rights reserved.
+// Copyright 2020 The Flutter Authors. All rights reserved.
```
Comment (Hixie) [inline LICENSE:2]: remove line 1 while you're at it, it's redundant.

Comment (domesticmouse) [reply LICENSE:1]: Line 1 removed.

**Pull request merged**